src/TemplatePath.php

```diff
@@ -6,11 +6,9 @@
 
 class TemplatePath
 {
-    /** @var string */
-    protected $path;
+    protected string $path;
```

Comment:
Php 7.3 is not typed property yet, also update to typed property should only for private modifier as can cause bc break for protected and public even on php 7.4 if the version change is minor, also, this is out of the PR scope.

Reply:
Done.